topics/proteomics/tutorials/proteome_annotation/tutorial.md

```diff
@@ -0,0 +1,310 @@
+>        - *"Column IDs: `c4`
+>        - *"Does file contain header"*: `Yes`
+>    - Numerous information can be extracted from the HPA source files, you can read user documentation at the end of the submission form of the tool for more detailed description. In this activity, in *"RNAseq/Ab-based expression data"*, we *"Select information to add to your list"*:
+>       - `Gene name`
+>       - `Gene description`
+>       - `RNA tissue category (according to HPA)`
+>       - `RNA tissue specificity abundance in "Transcript Per Million`
+>
+>
+>    > ### {% icon comment %} Outputs
+>    >
+>    > - **Add expression data on data 6**: Four columns were added (n°5, 6, 7 and 8) corresponding to the retrieved information from HPA.
+>    {: .comment}
+>
+>   > ### {% icon comment %} Comments
+>   >
+>   > Scroll down the table, note at the end of the list (column n°8), that **AMY1B**, **CALML5**, **PIP**, **ZG16B**, **CST4**, **MUC7**, **CST1** and **CST2** have been reported as highly enriched in salivary gland with elevated RNA transcript specific TPM value for each, suggesting that these proteins may come from the saliva and not from the exhaled breath condensate. We thus will remove these biological contaminants from our initial protein set.
+>   {: .comment}
+>
+{: .hands_on}
+
+
+> ### {% icon hands_on %} Hands-on: Filter the data to remove the biological contaminants (i.e. proteins highly expressed in salivary glands)
+>
+>  **Filter by keywords and/or numerical value** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `Add expression data on data 6` (output of **Add expression data** {% icon tool %})
+>    - Keep default option `Yes` for *"Does file contain header?"*
+>    - *"Operation"*: `Discard`. What we want is to remove biological contaminants.
+>    - Keep the `OR` option (by default) for the **operator** parameter. We don't need that parameter for a single filter.
+>    - First add a  {% icon param-repeat %} *"Insert Filter by keywords"* box with a list of keywords to be filtered out.
+>   In this step, we will filter out the lines that contain "salivary" in the column of RNA transcript specific TPM.
+>    - *"Column number on which to apply the filter"* : `c8`, the column of RNA transcript specific TPM
+>    - *"Search for exact match ?" : `No`
+>    - To *"Enter keywords"*, you can either copy and paste list of keywords to the text area or choose a file that contains keywords in text format, in which each lines contains a keyword. Here we choose to `copy/paste` the keyword to be filtered out :  `salivary`
+>
+>    > ### {% icon comment %} Outputs
+>   > Two output files are created:
+>   > - **Filtered Add expression data on data 6 - Discarded lines** (12 proteins)
+>   > - **FilteredAdd expression data on data 6** (151 proteins)
+>{: .comment}
+>
+> > ### {% icon tip %} Tip
```

Comment:
This box will be collapsed by default, so probably nice to add in title what tip is about, something like: 

```suggestion
> > ### {% icon tip %} Tip: Using genes instead of keywords
```